_posts/2020/2020-12-10-how-to-handle-class-names.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: How to handle class names
+date: 2020-12-01
+header_image: private/class-names.jpg
+header_position: center
+header_overlay: true
+category: coding
+tags: ["coding", "challenge", "devops", "css"]
+authors: ["Anne"]
+about_authors: ["azimmermann"]
+---
+
+A clean and arranged CSS structure is hard to maintain in growing projects. Therefore it is important to think right from the beginning how to structure your classes to not lose control. But if you start to search for methods and techniques to organize your CSS, you will soon realize that there are many ways to reach your goal!
+
+## The universe of the most popular CSS methodologies
+
+* [Object-Oriented CSS (OOCSS)](https://de.slideshare.net/stubbornella/object-oriented-css){:target="_blank"} _by Nicole Sullivan, introduced 2009_ 
+* [Block, Element, Modifier (BEM)](https://en.bem.info/){:target="_blank"} _by the developer team at Yandex 2005, open source since 2010_
+* [Atomic CSS](https://acss.io/){:target="_blank"} _by Thierry Koblenz, introduced 2013_
+* [Multilayer CSS (MCSS)](https://operatino.github.io/MCSS/en/){:target="_blank"} _by Robert Haritonov, introduced 2014_
+* [SUIT CSS](http://suitcss.github.io/){:target="_blank"} _by Nicolas Gallagher, introduced 2014_
+* [Attribute Modules for CSS (AMCSS)](https://amcss.github.io/){:target="_blank"} _by Glen Maddern & Ben Schwarz, introduced 2014_
+* [Scalable and Modular Architecture for CSS (SMACSS)](http://smacss.com/){:target="_blank"} _by Jonathan Snook, introduced 2015_
+* [Flat hierarchy of selectors (FUN / Enduring CSS)](https://benfrain.com/enduring-css-writing-style-sheets-rapidly-changing-long-lived-projects/#l7){:target="_blank"} _by Ben Frain, introduced 2015_
+* [Systematic CSS](https://www.yumpu.com/en/document/read/47573458/systematic-css){:target="_blank"} _by Kieran Potts, introduced 2015_
+* [Reasonable System for CSS Stylesheet Structure (RSCSS)](https://rscss.io/){:target="_blank"}** _by Denis Vieira, introduced 2017_
+* [CUBE CSS](https://piccalil.li/blog/cube-css/){:target="_blank"} _by Andy Bell, introduced 2020_
```

Comment:
I think providing an overview of some of the different approaches to structuring CSS is a good idea. 👍 

However, I think this list is a bit overwhelming. I think most readers will have a hard time trying to understand all those bullet points. And as a result, they probably won't click on the links either.

Maybe you can focus this list on the most essential items or even integrate them in a sentence or two, instead.